test/development/acceptance/ReactRefreshLogBox-builtins.test.ts

```diff
@@ -59,6 +59,9 @@ describe.each(['default', 'turbo'])('ReactRefreshLogBox %s', () => {
       expect(await session.getRedboxSource()).toMatchInlineSnapshot(`
         "./node_modules/my-package/index.js:1:1
         Module not found: Can't resolve 'dns'
+        > 1 | const dns = require('dns')
```

Comment:
what's difference between https://github.com/vercel/next.js/pull/64228/files#diff-05b6ff60dd6730bcd70eb57fc3313401ed96c5e427468d2aa2758894541e2c98L57 one removes existing stack and one adds new?

Reply:
This brings Next.js+Webpack in line with the Turbopack behavior, showing these frames (but not long bundled internal files)